deps/exokit-bindings/videocontext/include/VideoMode.h

```diff
@@ -3,14 +3,186 @@
 
 #include "VideoCommon.h"
 
+#include <cassert>
+
+
+extern "C" {
+#include <libavutil/avstring.h>
+#include <libavcodec/avcodec.h>
+#include <libavdevice/avdevice.h>
+#include <libswscale/swscale.h>
+}
+
 namespace ffmpeg {
 
+class VideoCamera
+{
+public:
+  AVFormatContext* pFormatCtx;
+  int videoStream;
+  AVCodec* pCodec;
+  AVFrame* pFrame;
+  AVFrame* pFrameRGB;
+  AVPacket packet;
+
+
+  VideoCamera(AVFormatContext* formatContext, AVCodec* codec, int videoStream)
```

Comment:
Couldn’t these definitions be in the cpp? That’s the convention everywhere else I think.